apstra/api_anomalies.go

```diff
@@ -224,3 +227,113 @@ func (o *Client) getAnomalies(ctx context.Context) ([]Anomaly, error) {
 	}
 	return result, nil
 }
+
+// also available in scotch/schemas/alerts.py:
+//   BGP_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {                 // node
+//   BLUEPRINT_RENDERING_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, { // node
+//   CABLING_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {             // node
+//   CONFIG_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {              // node
+//   CONFIG_MISMATCH_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {     // node
+//   DEPLOYMENT_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {          // node
+//   EXTENSIBLE_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {          // ?
+//   HOSTNAME_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {            // node
+//   INTERFACE_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {           // node
+//   LAG_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {                 // ?
+//   LIVENESS_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {            // node
+//   MAC_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {                 // ?
+//   MLAG_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {                // ?
+//   ROUTE_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {               // node
+//   STREAMING_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {           // ?
+//   PROBE_ANOMALY_SCHEMA = t.Object(BASE_ANOMALY_SCHEMA, {               // node
+
+type BlueprintAnomaly struct {
+	Id             ObjectId   `json:"id"`               // part of base schema
+	LastModifiedAt *time.Time `json:"last_modified_at"` // part of base schema
+	Severity       string     `json:"severity"`         // part of base schema
+	AnomalyType    string     `json:"anomaly_type"`     // part of base schema
+
+	Actual    json.RawMessage `json:"actual"`    // universal (near universal?)
```

Comment:
what does universal (nearly universal) mean?

Reply:
per teams chat: this was just a comment to myself indicating how often I saw each attribute in the python extensible attribute schema.